### DISCUSSION TITLE
Feature request - New window for AIO Logs

### DISCUSSION THREAD

**Discussion by rspierenburg** [nextcloud/all-in-one#3881] [Ideas] [answered]
Just had a quick question / comment.  Would it be possible to have a new window open in the browser when viewing logs from the AIO Interface?

**Comment by szaimen** [comment, marked as answer]
Hi, why would you need this?

**Reply by rspierenburg** [reply]
Mostly for a comfort feature.  But I find if I check one of the logs from the AIO interface my tendencies is to close the tab with the log rather than pressing the back button, this then closes the AIO interface in which case I need to reopen it.

**Reply by szaimen** [reply]
@marcoambrosini what do you think about this idea? Would you agree that logs should open in a new tab?

**Reply by marcoambrosini** [reply]
I think so yes, the main page would auto refresh when things are done right?

**Reply by szaimen** [reply]
Will be done in https://github.com/nextcloud/all-in-one/pull/3914

**Reply by rspierenburg** [reply]
This is awesome thanks!

**Reply by szaimen** [reply]
This is now released with v7.9.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
